src/GerryChain.jl

```diff
@@ -33,18 +33,17 @@ flip_chain,
 DistrictAggregate,
 DistrictScore,
 PlanScore,
+CompositeScore,
```

Comment:
I think `CompositeScore` can be public because there are possible use cases where the user wants to modify some state object but run multiple scoring functions. I don't know what they are, but I don't see the harm in exposing this to the user.

Reply:
alright then, fine by me!